src/server/router/peaRouter/peaRouter.ts

```diff
@@ -165,7 +165,7 @@ peaRouter.post('/updateServerSettings', asyncHandler(async (req: Request, res: R
 	const manager: ModularPlantManager = req.app.get('manager');
 	try{
 		manager.updateServerSettings(req.body);
-		res.status(200).send('"'+'Success!'+'"');
+		res.status(200).send('"'+'Successfully updated the server settings!'+'"');
 	} catch(e){
 		res.status(500).send(e.toString());
```

Comment:
Whats about the error? sorry didnt split into two comments

Reply:
In case of error, the catch-block will handle it. As you can see, the error will be sent as response with error code 500. 
Is this not how you want it?

The error will be shown in the snackbar in frontend.

<img width="1012" alt="Bildschirmfoto 2021-07-22 um 11 14 59" src="https://user-images.githubusercontent.com/62188772/126616106-aa5f2d1c-9af8-4039-b2a7-55d0977ca14e.png">